js/app.js

```diff
@@ -1,207 +1,82 @@
-  }
 };
 
-center.putTheNumbersInTheBroswer();
-
-var hill= {
-  name: 'Hill',
-  minCustomers: 23,
-  maxCustomers: 65,
-  avgCookies: 6.3,
-  customersEachHour: [],
-  peopleEveryHour: function() {
-    for (var i = 0; i < timeHours.length; i++) {
-      var customerForOneHour =  Math.floor(Math.random() * (this.maxCustomers - this.minCustomers + 1)+ this.minCustomers);
-      this.customersEachHour.push(customerForOneHour);
-    }
-  },
-
-  totalCookiesSoldToday: [],
-  addedTotalCookies: 0,
-
-  totalSold: function() {
-    for (var i=0; i < timeHours.length; i++) {
-      var totalCookiesSold = this.customersEachHour[i] * this.avgCookies;
-      this.totalCookiesSoldToday.push(Math.floor(totalCookiesSold));
-      this.addedTotalCookies = this.totalCookiesSoldToday[i] + this.addedTotalCookies;
-    }
-  },
+pike.putTheNumbersInTheBroswer();
 
-  putTheNumbersInTheBroswer: function() {
-    var hilllist = document.getElementById('hill');
-    this.peopleEveryHour();
-    this.totalSold();
-    for (var i = 0; i < timeHours.length; i++) {
-      var liEL = document.createElement('li');
-      liEL.textContent = timeHours[i] + ': ' + this.totalCookiesSoldToday[i] + ' cookies';
-      hilllist.appendChild(liEL);
-    }
-    liEL = document.createElement('li');
-    liEL.textContent = 'total sold: ' + this.addedTotalCookies;
-    hilllist.appendChild(liEL);
   }
+  tableContent.appendChild(trEL);
+  tdEL = document.createElement('td');
+  tdEL.textContent = this.addedTotalCookies;
+  trEL.appendChild(tdEL);
 };
 
-hill.putTheNumbersInTheBroswer();
-
-var alki= {
-  name: 'alki',
-  minCustomers: 23,
-  maxCustomers: 65,
-  avgCookies: 6.3,
-  customersEachHour: [],
-  peopleEveryHour: function() {
-    for (var i = 0; i < timeHours.length; i++) {
-      var customerForOneHour =  Math.floor(Math.random() * (this.maxCustomers - this.minCustomers + 1)+ this.minCustomers);
-      this.customersEachHour.push(customerForOneHour);
-    }
-  },
-
-  totalCookiesSoldToday: [],
-  addedTotalCookies: 0,
-
-  totalSold: function() {
-    for (var i=0; i < timeHours.length; i++) {
-      var totalCookiesSold = this.customersEachHour[i] * this.avgCookies;
-      this.totalCookiesSoldToday.push(Math.floor(totalCookiesSold));
-      this.addedTotalCookies = this.totalCookiesSoldToday[i] + this.addedTotalCookies;
-    }
-  },
-
-  putTheNumbersInTheBroswer: function() {
-    var alkilist = document.getElementById('alki');
-    this.peopleEveryHour();
-    this.totalSold();
-    for (var i = 0; i < timeHours.length; i++) {
-      var liEL = document.createElement('li');
-      liEL.textContent = timeHours[i] + ': ' + this.totalCookiesSoldToday[i] + ' cookies';
-      alkilist.appendChild(liEL);
-    }
-    liEL = document.createElement('li');
-    liEL.textContent = 'total sold: ' + this.addedTotalCookies;
-    alkilist.appendChild(liEL);
+function headerRow () {
+  var total = document.createElement('th');
+  var blankData = document.createElement('th');
```

Comment:
This works fine, but it is better practice to create the element immediately using it.